libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -85,6 +86,25 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // @Values: -1:Disabled,0:Bus0:1:Bus1
     // @User: Advanced
     AP_GROUPINFO("EXT_BUS", 7, AP_Baro, _ext_bus, -1),
+
+    // @Param: SPEC_GRAV
```

Comment:
I am considering adding a flags field to parameter to allow you to mark a parameter as being specific to a particular vehicle type, I think that would make some of this clearer